core/tauri/src/scope/fs.rs

```diff
@@ -252,6 +262,21 @@ impl Scope {
       false
     }
   }
+
+  /// This removes any paths added to allowed and forbidden scope by resetting
+  /// to the scope state to the given FsScope config
+  pub fn reset<R: crate::Runtime, M: crate::Manager<R>>(
+    &self,
+    manager: &M,
+    scope: &tauri_utils::config::FsScope,
+  ) -> crate::Result<()> {
```

Comment:
instead of taking scope as an argument, you could provide a `clear` method which will empty allowed and forbidden hash sets, and `reset` which only takes one argument `Manager<R>` to reset to default